csharp/Platform.Collections/Segments/Segment.cs

```diff
@@ -9,35 +9,73 @@
 
 namespace Platform.Collections.Segments
 {
+    /// <summary>
+    /// <para>Represents the segment <see cref="IList"/>.</para>
+    /// <para>Представляет сегмент <see cref="IList"/>.</para>
+    /// </summary>
+    /// <typeparam name="T"><para>The segment elements type.</para><para>Тип элементов сегмента.</para></typeparam>
     public class Segment<T> : IEquatable<Segment<T>>, IList<T>
     {
+        /// <summary>
+        /// <para>Returns the original list (of which this segment is a part).</para>
+        /// <para>Возвращает исходный список (частью которого является этот сегмент).</para>
+        /// </summary>
         public IList<T> Base
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get;
         }
+        /// <summary>
+        /// <para>Returns the offset of the relative source list (the index at which this segment starts).</para>
+        /// <para>Возвращает смещение относительного исходного списка (индекс с которого начинается этот сегмент).</para>
+        /// </summary>
         public int Offset
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get;
         }
+        /// <summary>
+        /// <para>Returns the length of a segment.</para>
+        /// <para>Возвращает длину сегмента.</para>
+        /// </summary>
         public int Length
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get;
         }
 
+        /// <summary>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class, using the passed list as the original, <paramref name="offset"/> segment and its <paramref name="length" />.</para>
```

Comment:
`the passed list as the original` ⇾ `the <paramref name="base"/> list`
`<paramref name="offset"/> segment` ⇾ `<paramref name="offset"/> of the segment`